sssoja/tv-bland
# fetches api data

### What changes have you made?

- Made two async requests from the `index` and `[show]` pages in order to display incoming data from the API
- Passed data to all components where dynamic data is required 

Note. no data was found for `Cast` so placeholder items have been used

<img width="463" alt="Screenshot 2020-11-28 at 00 52 20" src="https://user-images.githubusercontent.com/53219789/100489376-27b1c200-3114-11eb-866c-e1168b9acec1.png">


### Which issue(s) does this PR fix?

Fixes #9 

### Screenshots (if there are design changes)
<img width="249" alt="Screenshot 2020-11-28 at 00 06 35" src="https://user-images.githubusercontent.com/53219789/100488286-9b9c9c00-310d-11eb-92c0-dbfa26f5b775.png">

<img width="249" alt="Screenshot 2020-11-28 at 00 07 18" src="https://user-images.githubusercontent.com/53219789/100488303-b2db8980-310d-11eb-901b-8c12e58846d2.png">

### How to test

Open http://localhost:3000/ and click on a few shows at random to see if its information displays correctly
